lib/api/controllers/debugController.ts

```diff
@@ -117,110 +49,25 @@ export class DebugController extends NativeController {
    * Connect the debugger
    */
   async enable() {
-    if (this.debuggerStatus) {
-      return;
-    }
-
-    this.inspector.connect();
-    this.debuggerStatus = true;
-
-    for (const module of this.modules) {
-      await module.init(this.inspector);
-
-      for (const methodName of module.methods) {
-        if (!module[methodName]) {
-          throw new Error(
-            `Missing implementation of method "${methodName}" inside DebugModule "${module.name}"`
-          );
-        }
-        this.kuzzlePostMethods.set(
-          `Kuzzle.${module.name}.${methodName}`,
-          module[methodName].bind(module)
-        );
-      }
-
-      for (const eventName of module.events) {
-        module.on(eventName, async (payload) => {
-          if (!this.debuggerStatus) {
-            return;
-          }
-
-          const event = `Kuzzle.${module.name}.${eventName}`;
-          await this.notifyGlobalListeners(event, payload);
-
-          const listeners = this.events.get(event);
-          if (!listeners) {
-            return;
-          }
-
-          const promises = [];
-          for (const connectionId of listeners) {
-            promises.push(
-              this.notifyConnection(connectionId, DEBUGGER_EVENT, {
-                event,
-                result: payload,
-              })
-            );
-          }
-
-          // No need to catch, notify is already try-catched
-          await Promise.all(promises);
-        });
-      }
-    }
+    global.kuzzle.ask("core:debugger:enable");
```

Comment:
This should be awaited I think

Reply:
Every "Ask" call you described are sync, but for the sake of every other ask being awaited I will add the await keyword, this way there will not be any issue if somehow the ask functions are changed to async